runtime/doc/builtin.txt

```diff
@@ -8141,15 +8146,21 @@ py3eval({expr})						*py3eval()*
 
 		Can also be used as a |method|: >
 			GetExpr()->py3eval()
+			b",".join(l)'->py3eval({'l': ['a', 'b', 'c']})
```

Comment:
I suppose you are missing a leading single quote here?
```suggestion
			'b",".join(l)'->py3eval({'l': ['a', 'b', 'c']})
```

Reply:
Yes, done.